src/test/java/org/folio/marccat/resources/FixedFieldCodesGroupTest.java

```diff
@@ -8,31 +11,22 @@
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringRunner;
 
-import java.util.Map;
-
-import static io.restassured.RestAssured.given;
-
 @RunWith(SpringRunner.class)
 @ActiveProfiles("test")
 public class FixedFieldCodesGroupTest extends TestBase {
 
   private static final String FIXED_FIELDS_CODE_GROUPS_URL = "/marccat/fixed-fields-code-groups";
+  private static final String AUT_FIXED_FIELDS_CODE_GROUPS_URL = "/marccat/auth-fixed-fields-code-groups";
+  private static final String AUTHORITY_LEADER = "00215nz   2200097n  4500";
 
   @Test
   public void getFixedFieldsByLeader_return200Status() {
 
     String url = getURI("/marccat/fixed-fields-code-groups-by-leader");
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
 
-    given()
-      .param("leader", "01657nam a22002057i 4500")
-      .param("code", "008")
-      .param("lang", "ita")
-      .headers(headers)
-      .when()
-      .get(url)
-      .then()
-      .statusCode(200);
+    given().param("leader", "01657nam a22002057i 4500").param("code", "008").param("lang", "ita").headers(headers)
```

Comment:
IMO The old format is preferred

Reply:
Please, keep the old formatting about tests and APIs.

Reply:
I have changed them